recipes/r-arrow/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = '0.12.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-arrow
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: e93e43343544e344bbc912b89da01d8abf66596f029d26b2b135b102a9e39895
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: True  # [win]
```

Comment:
What are the problems this is skipped on Win?

Reply:
@xhochy Good question: Tried to build on windows 2-3 times before opening the PR, which was not successful, see e.g. https://ci.appveyor.com/project/dbast/staged-recipes/builds/23117017

Then I had a look into the upstream repo and found no indication that this was ever build on windows (./r excluded in appveyor.yml ... nothing windows related in ./r/src/configure and ./r/README.md, all about macOS / various Linux OSs... no ./r/src/Makewars.win etc.) -> Decided to skip for now until there is something upstream ... 

So we can keep the skip for now and activate windows later in the feedstock or do it here, if there is some confidence that this works with the 0.12.1 src tarball ...

Reply:
Oh, sorry. Forgot that we're still missing this upstream

Reply:
ok. thanks for reviewing and confirming!